src/jti_background.js

```diff
@@ -161,11 +174,24 @@ function fetchDefaultTemplates (callback) {
     });
 }
 
+function getData (callback) {
+    chrome.storage.sync.get(StorageID, function (templates) {
+        if (templates[StorageID]) {
+            var templateJSON = templates[StorageID];
+            templateJSON.templates = templateDataToJSON(templateJSON.templates);
+            templateJSON.options.domains = domainDataToJSON(templateJSON.options.domains);
+            templateJSON.options.inputIDs = inputIDDataToJSON(templateJSON.options.inputIDs);
```

Comment:
While it looks like we only do this once (data -> JSON) it would make sense to have a matching function to JSONtoData -> dataToJSON and use it here.

Reply:
Right, that's why I didn't think of refactoring it but I'll make a function called JSONToData